### DISCUSSION TITLE
How init a discussion on my repository?

### DISCUSSION THREAD

**Discussion by gusttavonl** [Schweinepriester/github-profile-achievements#576] [Q&A] [answered]
How init a discussion on my repository? it is possible?

**Comment by LarsPeterson** [comment, marked as answer]
@GustavoNoronha0

![image](https://user-images.githubusercontent.com/25396567/213301341-db5f014e-55fc-4e64-9160-3eb317e14fa5.png)

![image](https://user-images.githubusercontent.com/25396567/213301378-fe818dca-20da-430e-9659-517b322804e1.png)

![image](https://user-images.githubusercontent.com/25396567/213301407-b52b414e-68fa-44e0-801f-db8392b02c5b.png)

1. Go to "Settings" tab in your repo.
2. On "General" tab, scroll down to "Features"
3. Check "Discussions"

**Reply by gusttavonl** [reply]
Thank you!

**Reply by LarsPeterson** [reply]
@GustavoNoronha0 :)
